packages/valory/skills/proposal_collector_abci/behaviours.py

```diff
@@ -222,6 +222,9 @@ def _get_updated_proposals(self) -> Generator[None, None, str]:
                 f"Retrieving proposals from Tally API [{self.params.tally_api_endpoint}]. Request variables: {variables}"
             )
 
+            # Wait for a couple seconds to avoid 429
+            yield from self.sleep(2)
```

Comment:
Let's make this configurable, or at least create a constant.

Reply:
Sorted